src/content/docs/en/welcome.mdx

```diff
@@ -76,6 +76,86 @@ As of now, the only data we collect is a count of how many times a package has b
+- Usability
+  - You never need to touch the command line.
+    - If there are cases that require the command line, and those situations might arise to other normal users too, you should [report this bug](https://github.com/Ultramarine-Linux/ultramarine).
+  - Ultramarine has sane defaults.
+    - Popular programs that you probably need are installed by default, including LibreOffice, Firefox, drivers, etc.
+    - We include repositories that provide popular software.
+  - There are many more apps you can installed from an app store. They are verified carefully by dedicated teams.
+    - The Fedora and RPM Fusion repositories are used and trusted by millions.
+    - Terra is reviewed by Fyra Labs and the Ultramarine Project. You can also [take a look at our security policy](https://github.com/terrapkg/packages/blob/f39/SECURITY.md).
+    - Flathub is used by users across the entire Linux/BSD ecosystem. If you install apps from an app store, you are most likely installing apps from Flathub, which are isolated from your system environment when run, making attacks less likely to succeed.
+
+### Why Ultramarine (instead of Windows)
+
+- Performance
+  - Windows bloat your computer with many different potentially unwanted programs that slow down your computer. Ultramarine does not.
+  - Ultramarine provides a few lightweight desktop environments (namely Budgie/Flagship and XFCE), which run faster and provides a snappier experience.
+  - Ultramarine uses newer technologies that are not available on Windows due to backwards compatibility. These technologies brings better performance.
+    - For example, task schedulers, file systems, shells, etc.
+    - Most modern servers (including giant web servers) use Linux simply because it's much faster.
+- Privacy
+  - We do not collect/track your private data ever because that's a waste of storage.
+  - We don't even know how many Ultramarine devices there are (unlike some companies).
+  - When you setup Windows, you need to accept a very long license. Well, we don't even have a license agreement page in our installer.
+  - Ultramarine is open source. Anyone can check if we track/collect your data by reviewing the source code. We don't.
+- Security
+  - Security updates are much quicker than Windows.
+    - Updates don't require reboots.
+  - It's much more unlikely for malware, viruses, etc. to target Linux due to its current low popularity.
+  - Our open source model makes it very easy for anyone to spot bugs/vulnerabilities and fix them quicker.
+- Stability
+  - Ultramarine uses a stable release cycle. Updates will not break your computer.
+  - When there is a chance a system upgrade could break your computer, we warn you about it.
+    - We explicitly advice people to contact support to check if the upgrade could actually break the system.
+    - Our support team can help you solve any potential issues when you would like to perform a system upgrade.

```

Comment:
"while performing a system upgrade" instead of "when you would like to perform a system upgrade"